Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -869,6 +876,23 @@ final class RecoveryTests: XCTestCase {
     )
   }
 
+  func testRecovery64c() {
+    assertParse(
+      """
+      private var a 1️⃣b : Int = ""
+      """,
+      diagnostics: [
+        DiagnosticSpec(
+            locationMarker: "1️⃣",
+            message: "unexpected code 'b' before type annotation"
+        )
+      ],
+      fixedSource: """
+        private var a b : Int = ""
```

Comment:
Should we in this case merge `a` and `b`

Reply:
I think that would make sense.

Reply:
Fixed and rebased in main